src/engines/kotor/module.cpp

```diff
@@ -515,27 +515,33 @@ void Module::handlePCMovement() {
 		float x, y, z;
 		_pc->getPosition(x, y, z);
 		float yaw = SatelliteCam.getYaw();
+		float newX, newY;
 
 		if (_forwardBtnPressed && !_backwardsBtnPressed) {
 			_pc->setOrientation(0, 0, 1, Common::rad2deg(yaw));
-			movePC(x - PC_MOVEMENT_SPEED * sin(yaw) * _frameTime,
-			       y + PC_MOVEMENT_SPEED * cos(yaw) * _frameTime,
-			       z);
+			newX = x - PC_MOVEMENT_SPEED * sin(yaw) * _frameTime;
+			newY = y + PC_MOVEMENT_SPEED * cos(yaw) * _frameTime;
```

Comment:
I missed that for the previous PR, but in xoreos, we prefer `kPCMovementSpeed` over `PC_MOVEMENT_SPEED`. It should also be a `static const`, not just a `const`. Same goes for `ROTATION_SPEED` in satellitecamera.cpp.